polar-wasm-api/Makefile

```diff
@@ -0,0 +1,10 @@
+.PHONY: build test
+
+build:
+	wasm-pack build --dev --target nodejs
+	rm pkg/.gitignore pkg/package.json
+	cp --recursive pkg/. ../languages/js/lib
+	rm -rf pkg
+
+test:
+	wasm-pack test --node
```

Comment:
If they also pass on a headless browser might be worth just running those tests now too to make doing the browser javascript api easier later. If they don't it'll take work anyway and we can just do that later.

Reply:
They pass in Chrome on my machine -- I'll try running 'em in CI and see what happens. Firefox tests work but not headless, and I don't have Safari on this machine.

Reply:
On second thought, after a bit of googling I can't figure out how to run the same set of tests on Node + a browser since (AFAICT) there's a special macro to alert wasm-pack that they should be run in a browser:

```rs
wasm_bindgen_test::wasm_bindgen_test_configure!(run_in_browser);
```

and I don't really feel like having two identical copies of the same test files, one with that macro and one without, which is all my tired brain can come up with right now.

We can circle back when browser support becomes necessary.